library/Neovim.hs

```diff
@@ -153,11 +153,11 @@ just keep reading. @:-)@
 {- $existingplugins
 The easiest way to start is to use the stack template as described in the
 @README.md@ of this package. If you initialize it in your neovim configuration
-directory (@~/.convig/nvim@ on linux-based systems), it should automatically be
```

Comment:
Looking at [the docs on Hackage](https://hackage.haskell.org/package/nvim-hs-2.1.0.4/docs/Neovim.html#g:3), this renders as `~.convignvim`. I don't think this will fix that; is that just happening for me or what's the reason it doesn't show the `/`s on Hackage?

Reply:
The slashes also have to be escaped, e.g. `@~\/.convig\/nvim@`

Reply:
Ah okay, that makes sense. Thank you! If you wouldn’t mind sharing, is there somewhere I can read about the formatting of Haskell/Hackage docs? I’m new to the way Haskell doc comments are formatted.